### ISSUE TITLE
Platform table filter - opening filter options with VoiceOver does not work properly

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12785]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

1. Create some platform table with sorting and filtering. It may be copy of [example from documentation](https://sap.github.io/fundamental-ngx/#/platform/table/preserved-state).
2. Turn on VoiceOver (MacOs version 14.7) 
3. Open  'Filter' using VoiceOver you should see filter dialog with 3 options 'price', 'status' and 'status color'.
4. Try to open 'Status color' using only VoiceOver. 

Dialog presents filter options for 'Status' not 'Status color'.
